src/Data/Nat/Divisibility.agda

```diff
@@ -125,42 +165,43 @@ module ∣-Reasoning where
 
   open ∣-syntax _IsRelatedTo_ _IsRelatedTo_ ∣-go public
 
+open ∣-Reasoning
```

Comment:
I know this is tempting, but bitter experience has always shown me that after you make this change, it'll only be a matter of weeks before someone tries to add something that needs a different form of reasoning and you'll have to revert the change again.

That nugget of wisdom should probably be added to the style guide...

Reply:
Ha! Sigh... I suspect you are correct. Indeed I think the style guide does already say something about making appeals to `Reasoning` modules local to any particular lemma... and I thought I could get away with it otherwise. Oh dear!

Reply:
Thanks, sorry!